steps/id-and-residency/carer.tsx

```diff
@@ -131,7 +131,7 @@ const step: ProcessStepDefinition<ResidentDatabaseSchema, "carer"> = {
   },
   step: {
     slug: PageSlugs.Carer,
-    nextSlug: PageSlugs.Verify,
+    nextSlug: PageSlugs.OtherSupport,
```

Comment:
you'll need to wrap this in `nextSlugWithId` (like you don in the `other-support`'s `nextSlug`, as this passes the id through in the URL